### DISCUSSION TITLE
Ubuntu22.04 product

### DISCUSSION THREAD

**Discussion by adrian-fej** [uyuni-project/uyuni#6546] [Devel]
Hello,
I am trying to get Ubuntu 22.04 product in order to be able to update it. I don't know what am I doing wrong but as installed products I have unknown. I have registered and  can see only Ubuntu16 and Ubuntu18 as products

**Comment by ipanmeri** [comment]
Hi there,
I just finished a fres install of Uyuni on OpenSuse Leap 15.5 and wondering the same thing.
The only Ubuntu channels or products I find are

`[ ] ubuntu-16.04-pool for amd64 Ubuntu 16.04 [ubuntu-16.04-pool-amd64]
[ ] ubuntu-18.04-pool for amd64 Ubuntu 18.04 [ubuntu-18.04-pool-amd64]
`

How do I add 20.04 and 22.04?
Thanks

**Reply by ipanmeri** [reply]
https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/clients-ubuntu.html
